### ISSUE TITLE
Normal enemies

### ISSUE THREAD

**Issue by zockerhuhn** [Sub-game-team/Subgame#12]
Deal 2-5 points of damage.
Have very little health

**Comment by zockerhuhn** [comment]
not done

**Comment by zockerhuhn** [comment]
no @FlederJ

**Comment by FlederJ** [comment]
yes @zockerhuhn , because I guess you want me to add health, pathfinding and attack patterns to the enemy. Also, it is literally only one enemy and we won't have one normal enemy in the entire game.

**Comment by zockerhuhn** [comment]
The health will be part of the torpedo issues so don't worry about that. Pathfinding and attack patterns are part of this issue. New normal enemies should maybe be a new issue to avoid a lot of stuff in one issue
@FlederJ
